pkgs/servers/matrix-conduit/default.nix

```diff
@@ -1,14 +1,14 @@
-{ lib, rustPlatform, fetchFromGitLab, pkg-config, sqlite, stdenv, darwin, nixosTests, rocksdb_6_23 }:
+{ lib, rustPlatform, fetchFromGitLab, pkg-config, sqlite, stdenv, darwin, nixosTests, rocksdb }:
```

Comment:
Would be nice to unroll these.

```
{ lib
, rustPlatform
, fetchFromGitLab
, pkg-config
, sqlite
, stdenv
, darwin
, nixosTests
, rocksdb
}:
```

Reply:
Yeah I agree, I just didn't do it because I wasn't sure if people would be unhappy about the extra diff noise.